config/initializers/sidekiq.rb

```diff
@@ -26,7 +26,7 @@ def not_past_scheduled_time?(current_time)
   # https://github.com/ondrejbartas/sidekiq-cron/issues/254
   # Sidekiq default is 5, we don't need it quite that often but would like it more than
   # every 30 seconds which the gem defaults to
-  Sidekiq.options[:poll_interval] = 10
+  Sidekiq[:poll_interval] = 10
```

Comment:
actually I think it would be better with
```suggestion
  config[:poll_interval] = 10
```
since it's already in this block

Reply:
good callout, ty. will update.